### PR TITLE
feat(version-update): go to 3.2.2 and add nextjs

### PR DESCRIPTION
**Pull request by doralchan**
Logos still appear. Lookin good. Connected to https://github.com/getsentry/platformicons/pull/20

Screen Shot 2020-11-09 at 6 04 21 PM

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2707,9 +2707,9 @@
     moment ">=2.14.0"
 
 "@types/node@*":
-  version "13.5.0"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-13.5.0.tgz#4e498dbf355795a611a87ae5ef811a8660d42662"
-  integrity sha512-Onhn+z72D2O2Pb2ql2xukJ55rglumsVo1H6Fmyi8mlU9SvKdBk/pUSUAiBY/d9bAOF7VVWajX3sths/+g6ZiAQ==
+  version "14.14.7"
```
Comment (EvanPurkhiser) [inline yarn.lock:2710]: Did this intentionally bump our `@types/node` version?

Comment (doralchan) [reply yarn.lock:2710]: I don't know how this happened when all I did was try to update the platformicons package...

**Review by dashed — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (dashed) [reply yarn.lock:2710]: This is fine. This is only for TS types; which will not end up in the final bundle.

Comment (billyvg) [reply yarn.lock:2710]: We also don't directly depend on the package.

**Pull request closed without merging**